project/dexter-server/public/AdminSE/adminSECtrl.js

```diff
@@ -0,0 +1,85 @@
+adminSEApp.controller('adminSECtrl', function($scope, $http, $location ,$routeParams, $log) {
+    "use strict";
+
+    $('select').on('select2:select', function (e) {
+        addModulePathList(e);
+        changeSelectedModuleCount();
+    });
+
+    $('select').on('select2:unselect', function (e) {
+        removeModulePathList(e);
+        changeSelectedModuleCount();
+    });
+
+    function changeSelectedModuleCount(){
+        var count = $scope.selectedModulePathList.length;
+        $("#removeModuleCount").html(parseInt(count));
+        if(count > 0) {
```

Comment:
if 'count' value is always over than zero(>=0), you can do like below:
if(count > 0){
  ...
} else {
  ...
}